frontend/src/components/Header.tsx

```
@@ -0,0 +1,39 @@
+          <h1>RAG Processor</h1>
+        </div>
+
+        <div className="header-user">
+          {isLoading && <span className="loading-indicator">Loading...</span>}
+
+          {error && <span className="error-indicator">Auth Error</span>}
+
+          {!isLoading && !error && isAuthenticated && user && (
+            <div className="user-info">
+              <span className="user-email">{user.email}</span>
+              <button onClick={logout} className="logout-button">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)